src/java-runtime/assets/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
```

Comment:
Add CSP header

Reply:
Explicitly enabled 'unsafe-inline' to unblock this PR.